pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -37,7 +38,12 @@ func updateSuccess(r *NetworkInfoReconciler, c context.Context, o *v1alpha1.Netw
 	metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerUpdateSuccessTotal, common.MetricResTypeNetworkInfo)
 }
 
-func deleteSuccess(r *NetworkInfoReconciler, _ context.Context, o *v1alpha1.NetworkInfo) {
+func deleteSuccess(r *NetworkInfoReconciler, c context.Context, o *v1alpha1.NetworkInfo, client client.Client, staleVPCs []*model.Vpc, ncName string, allVPCs sets.Set[string]) {
+	staleVPCNames := sets.New[string]()
+	for _, vpc := range staleVPCs {
+		staleVPCNames.Insert(*vpc.DisplayName)
+	}
+	deleteVPCNetworkConfigurationStatus(c, client, ncName, staleVPCNames, allVPCs)
```

Comment:
can we move deleteVPCNetworkConfigurationStatus logic out of function: deleteSuccess
it will deleteSuccess func consistent with our status update functions, like updateSuccess
We can call deleteSuccess when deleteVPCNetworkConfigurationStatus is successful

Reply:
updated.